routers/api/v1/repo/pull.go

```diff
@@ -0,0 +1,418 @@
+// Copyright 2015 The Gogs Authors. All rights reserved.
```

Comment:
Right, I'm not entirely sure if we can just change it, since I'm not the original author of this... @lstahlman is IIRC